src/app/search/state/search.service.ts

```diff
@@ -285,6 +285,21 @@ export class SearchService {
     });
   }
 
+  setFacetAutocompleteTerms(hits: any) {
+    const autocompleteTerms = [];
+    for (let i = 0; i <= 3; i++) {
```

Comment:
Where does 3 come from? Suggest a const with a meaningful name

Reply:
Following the other autocomplete implementation where at most there are 4 terms